src/internal/document_api/mod.rs

```diff
@@ -685,6 +736,95 @@ fn process_users(
     (user_errs, users_with_key)
 }
 
+fn process_policy(
+    policy_result: &PolicyResult,
+) -> (Vec<DocAccessEditErr>, Vec<WithKey<UserOrGroup>>) {
+    let (de_errs, policy_eval_results): (Vec<DocAccessEditErr>, Vec<WithKey<UserOrGroup>>) =
```

Comment:
I'm not sure it's obvious what `de` stands for here. Something like `doc_edit_errs` is still pretty terse but easier to parse

Reply:
`de` is common in Rust for deserialization. The errors are the result of public keys that failed to deserialize. I'll rename to `pubkey_errs` and see if that makes more sense.